src/TLuaInterpreter.cpp

```diff
@@ -5110,7 +5101,8 @@ int TLuaInterpreter::playSoundFile( lua_State * L )
     {
         sound.replace('\\', "/");
     }
-    mudlet::self()->playSound( sound, soundVolume );
```

Comment:
I'm not sure if QMediaPlayer::Error would have populated yet since the return-to-lua would occur the next line immediately following the playSound request. It would likely still be in the process of finding/opening the sound file/ codecs.